config.yaml

```diff
@@ -20,17 +20,10 @@ sdk:
       #
       # Environment variable: OTEL_SERVICE_NAME
       service.name: !!str "unknown_service"
-  # Configure context propagators. Each propagator has a name and args used to configure it. None of the propagators here have configurable options so args is not demonstrated.
+  # Configure context propagators. Each propagator has a name used to configure it.
   #
   # Environment variable: OTEL_PROPAGATORS
-  propagators:
-    - name: tracecontext
-    - name: baggage
-    - name: b3
-    - name: b3multi
-    - name: jaeger
-    - name: xray
-    - name: ottrace
+  propagators: [tracecontext, baggage, b3multi]
```

Comment:
Unless there's a good reason, we should merge this with the copy in the root.

Reply:
i'm assuming you mean `json_schema/kitchen-sink.yaml` should be removed yah?